### ISSUE TITLE
Fuzz tool outputs WA MVP incompatible module

### ISSUE THREAD

**Issue by Kimundi** [WebAssembly/binaryen#1634]
Hi, I'm toying around with a custom implementation of the webassembly spec, and I discovered [this](https://github.com/WebAssembly/binaryen/wiki/Fuzzing) page while looking for some test suites /fuzzers for my code.

I've found the `wasm-opt --translate-to-fuzz` option to generate a random module helpful so far, but I'm running into one problem: It seems to emit more than just the MVP spec of webassembly. Specifically, I'm running into the issue that it emits shared memories - I get a binary sequence `01 03 01 01` for the memory section, where the second is byte is invalid but presumably means that the memory is shared.

Is there a way to disable this behaviour?

**Comment by Kimundi** [comment]
Update: I've discovered https://github.com/WebAssembly/binaryen/pull/1513, which was basically about the same issue, but seems to not have exposed the no-atomics option as a commandline flag.

**Comment by kripken** [comment]
Yeah, there isn't a commandline flag for those things, but you can edit src/tools/fuzzing.h and adjust whether to emit atomics (`emitAtomics`), as well as other things.

It would be nice to add commandline options for them all, if someone has the time.

**Comment by tlively** [comment]
We have command line feature flags that the fuzzer respects now.
